brave/brave-ios
# Fix #5984: Tab-Sync UI Revamp Part 2

## Summary of Changes

1. Tab List UI redesign

2. Sending Screen - animation with send

This pull request fixes #5984

## Submitter Checklist:


## Test Plan:

1. Test the new UI changes Tab Tray with devices from tabs and observe the behaviour is same

2. Send Web Site Animation

- Open a website
- Go settings menu
- Select Send to Your Devices
- Select a device and Press Send to Your Device
- Check screen with sending and sent is shown


## Screenshots:

- Tab Tray UI Changes

![send1 1](https://user-images.githubusercontent.com/6643505/190476698-ff0b1539-7370-4723-a1c0-ac1806beffc3.png)

https://user-images.githubusercontent.com/6643505/190476772-ba0b98e2-7d24-4914-a945-83bc9473a124.MP4


-Send Web Page

![send1 1:1](https://user-images.githubusercontent.com/6643505/190476747-1ea57236-7ce7-4941-87af-16afb8c66feb.png)


https://user-images.githubusercontent.com/6643505/190476843-6191a934-4ea2-4d09-ad2b-2fe4f9b8b04b.MP4



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
